apps/freelance/src/pages/Login/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { useTranslation } from "react-i18next";
+import { StyledPage, Container } from "./style";
+import { ThemeColors, FormContainer, MessageContainer } from "@freelance/components";
+import { ThemeProvider } from "styled-components";
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface LoginProps {}
```

Comment:
а они тут вообще когда то нужны будут? может его вообще убрать?

Reply:
I don't think it will be needed. I had it because I was following the example files, and I thought it was how it supposed to be, even if not been used. I will fix having it removed.